out/WIPL.asm

```diff
@@ -771,55 +747,49 @@ L_055a:
 055c:    c3 42        ld BL, (PC+0x42)
 055e:    49           sub! BL, AL
 055f:    14 02        bz L_0563
```

Comment:
Skips printing the char if the last byte written was '\r'?

I think this might be a version of printchar which eats any extra characters after the end of a line?